index.html

```diff
@@ -460,6 +472,26 @@ <h2>
             </li>
           </ol>
         </dd>
+        <dt>
+          <dfn>touchEvents</dfn>
```

Comment:
As above... I think it's incorrect to call these touch events. They are activation points on the track pad at some specific moment in time.